assets/app/views/browse/_build-details.html

```diff
@@ -7,6 +7,8 @@
         <dd>
           <status-icon status="build.status.phase"></status-icon>
           {{build.status.phase}}
+          <button class="btn btn-default build-status-button btn-xs" ng-click="cancelBuild(build, buildConfigName)" ng-if="!build.metadata.deletionTimestamp && (build | isIncompleteBuild)">Cancel</button>
+          <button class="btn btn-default build-status-button btn-xs" ng-click="cloneBuild(build.metadata.name)" ng-hide="build.metadata.deletionTimestamp || (build | isIncompleteBuild)" ng-disabled="!canBuild()">Rebuild</button>
```

Comment:
This button has moved to the primary actions for the build (outside of the tabs).